base/missing.jl

```diff
@@ -196,6 +217,17 @@ function iterate(itr::SkipMissing, state...)
     item, state
 end
 
+IndexStyle(::Type{<:SkipMissing{T}}) where {T} = IndexStyle(T)
+eachindex(itr::SkipMissing) =
+    Iterators.filter(i -> @inbounds(itr.x[i]) !== missing, eachindex(itr.x))
+keys(itr::SkipMissing) =
+    Iterators.filter(i -> @inbounds(itr.x[i]) !== missing, keys(itr.x))
+@propagate_inbounds function getindex(itr::SkipMissing, I...)
+    v = itr.x[I...]
+    v === missing && throw(MissingException("index $I points to a missing value"))
```

Comment:
Ok, this is pedantic, but I don't like the word points.  :)

```suggestion
    v === missing && throw(MissingException("the value at index $I is missing"))
```